src/posix/platform/netif.cpp

```diff
@@ -147,6 +147,9 @@ extern int
 #include "common/logging.hpp"
 #include "net/ip6_address.hpp"
 
+unsigned int gNetifIndex = 0;
+char         gNetifName[IFNAMSIZ];
```

Comment:
nit:
```suggestion
char         gNetifName[IFNAMSIZ] = "";
```

Reply:
@bukepo , can we resolve this comment?

Reply:
global array is already initialized to all zeros.